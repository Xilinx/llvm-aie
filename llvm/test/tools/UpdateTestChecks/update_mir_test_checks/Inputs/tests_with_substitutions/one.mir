# RUN: llc -mtriple=%triple -run-pass=none -o - %s | FileCheck %s

---
name:            test
body:             |
  bb.1:
    RET 0, implicit $eax
...
