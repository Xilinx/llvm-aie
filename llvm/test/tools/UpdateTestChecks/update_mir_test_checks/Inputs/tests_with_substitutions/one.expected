# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
# RUN: llc -mtriple=%triple -run-pass=none -o - %s | FileCheck %s

---
name:            test
body:             |
  bb.1:
    ; CHECK-LABEL: name: test
    ; CHECK: RET 0, implicit $eax
    RET 0, implicit $eax
...
